help.c

```diff
@@ -32,15 +30,21 @@
 #include <wchar.h>
 #include "globals.h"
 #include "keymap.h"
-#include "keymap_defs.h"
 #include "lib/lib.h"
 #include "mapping.h"
 #include "mbyte.h"
 #include "mutt_curses.h"
+#include "opcodes.h"
 #include "options.h"
 #include "pager.h"
 #include "protos.h"
 
+static const char *HelpStrings[] = {
+#define DEFINE_HELP_MESSAGE(opcode, help_string) [opcode] = help_string,
+  OPS(DEFINE_HELP_MESSAGE)
+  [OP_MAX] = NULL,
+};
```

Comment:
As in the other comment, I'd undef the temporary macro just after use.
Also, do you ever expect `opcodes` to come unordered or with gaps? If so, how? If not, why the explicit array index?